src/test/resources/keypair.txt

```diff
@@ -0,0 +1,4 @@
+publicKey : [B@6ab778a
```

Comment:
pay attention to this. it is not a public key, it is java object representation 
public key should be also 32 bytes long (64 bytes hexencoded value)